### PR TITLE
Adding Certificate Manager

### PR DESCRIPTION
**Pull request by surbhig12**
### Change Description ###

Restructured the content for Certificate Manager tool for Visual Studio. Please verify. 

To refer to the original content, see https://developer.tizen.org/development/visual-studio-tools-tizen/tools/certificate-manager. 

Please note: If there are any issues with the image size or resolution, it will be fixed after all the topics are completed.

### CODE REVIEW

File: docs/application/vstools/tools/certificate-manager.md [hunk 1]
```diff
@@ -1,104 +1,254 @@
...
->It is recommended to keep your certificates and password safe in the local repo to prevent it from being compromised.
+If you already have a certificate profile, you can import it or you can create a certificate profile.
 
-A certificate profile consists of an author certificate and 1 or 2 distributor certificates:
+### Import certificate profile
 
--   An author certificate includes information about the author of the application. It is used to create an author signature, which ensures the integrity of the application from the author since the publication of the application.
--   A distributor certificate includes information about the distributor of the application, such as a store. It is used to create a distributor signature, which ensures the integrity of the application from the distributor since the distribution of the application.
+1. In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Certificate Manager**.
+2. To import the profiles.xml file, click **Browse** .
```
Comment (surbhig12) [inline docs/application/vstools/tools/certificate-manager.md]: There i s an extra space after the word "Browse".
In the next commit, I will fix this.

File: docs/application/vstools/tools/certificate-manager.md [hunk 2]
```diff
@@ -1,104 +1,254 @@
...
+
+- **Author Certificate** includes information about the author of the application. It is used to create an author signature, which ensures the integrity of the application from the author since the publication of the application.
+- **Distributor Certificate** includes information about the distributor of the application, such as a store. It is used to create a distributor signature, which ensures the integrity of the application from the distributor since the distribution of the application.
+
+To distribute your application, you must create a certificate profile and sign the application with it. To create a certificate profile:
+
+1. Perform step 1 to step 3 under [import certificate profile](#import-certificate-profile).
+2. In the **Tizen Certificate Manager** window, click **App Signing**.
+
+   ![Tizen Certificate Manager window](media/cert_app_signing.PNG)
```
Comment (mijin85cho) [inline docs/application/vstools/tools/certificate-manager.md]: Just for the reference. When you capture the images, please use the Windows 10 PC. For now, Windows 10 is the environment guideline for the image capturing. I think you can also check that the updated images are in different UI compared to original images.

Comment (surbhig12) [reply docs/application/vstools/tools/certificate-manager.md]: @mijin85cho,

Sure, we will use Windows 10.

**Comment by surbhig12** [comment]
As per @journey2w's  advice, I am closing this PR for now.

**Pull request closed without merging**